src/App.css

```diff
@@ -1,39 +1,81 @@
-.App {
-  text-align: center;
+* {
+  box-sizing: border-box;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.my-app {
+  display: flex;
+  justify-content: center;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.calculator {
+  box-shadow: 0 10px 20px rgba(0, 0, 0, 0.1), 0 6px 6px rgba(0, 0, 0, 0.1);
+  margin-top: 8%;
+  width: 500px;
+  border: none;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.display {
+  background-color: rgb(14, 14, 41);
+  color: #fff;
+  height: 50px;
   display: flex;
-  flex-direction: column;
+  justify-content: end;
   align-items: center;
+  padding-right: 5px;
+  font-size: 24px;
+  font-weight: bold;
+}
+
+.operator-digits-wrapper {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  justify-content: flex-end;
+}
+
+.digits {
+  width: calc(75% - 5px);
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.digits button {
+  width: calc(33.33% - 2px);
+  height: 50px;
+  margin: 0 2px 2px 0;
+  display: flex;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  align-items: center;
+  font-size: 18px;
 }
 
-.App-link {
-  color: #61dafb;
+.operators {
+  width: 30%;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
+.operators button {
+  background-color: rgb(39, 163, 163);
+  height: 50px;
+  margin: 0 0 2px 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 18px;
+}
```

Comment:
## Your Design
![image](https://user-images.githubusercontent.com/93039359/226647820-de846551-bbf2-4ae7-8ecd-f227ac031d76.png)

## Required Design
![image](https://user-images.githubusercontent.com/93039359/226647992-44e547a6-6bb8-4927-af7c-cd85a5354c52.png)

- [ ] It would be nice if you make your project look similar to the Required design :+1: to make your project look professional :blush: It would be great if the coloring of the button should be similar to the design :+1: and secondly you have gaps between each button :eyes: kindly remove that gap as you can see in the design :blush: